### ISSUE TITLE
MacType Tuner > INI needs reworking

### ISSUE THREAD

**Issue by sammilucia** [snowie2000/mactype#551]
> Curiously, I noticed that in MacType Tuner it shows hinting being on

@ChicoThorn I think that's possibly a bug in MacType Tuner .... @snowie2000 I've noticed a few inconsistencies like this - do you think Tuner logic needs to be reworked / double check all new ini settings?

More info:
- Commented lines ";" are not preserved
- If you open an INI in MacType Tuner and save it, Tuner won't preserve any settings it doesn't understand
- e.g. the [Preview] section is not preserved
- any Experimental sections are not preserved
- Also "HintingMode=0" is changed to "1" on save
- If untouched, "NormalWeight", "BoldWeight", "EnableKerning", "RenderWeight" "GammaValue", "Contrast", "FontSubstitutes", and "LcdFilter" are changed on saving
- "TextTuning", "TextTuningR", "TextTuningG", "TextTuningB" are changed on saving
- "Saturation" is not preserved
- "LoadOnDemand" is not preserved
- "UseMapping" is not preserved"
- Possibly other problems

MacType (to me) has always been more about manually tweaking the INI files, so I don't see this as something that needs to be fixed for the 2019.1 release.

_Originally posted by @sammilucia in https://github.com/snowie2000/mactype/issues/518#issuecomment-503551183_

**Comment by ChicoThorn** [comment]
Thank you! That's all really helpful information... good to know! 😊

**Comment by snowie2000** [comment]
Currently, the tuner will fully rewrite the profile in the way it is defined in the code. It doesn't care about anything else it can't recognize, the same as what you observed.

And I agree that it should be remodeled. I would like to put it to the next milestone rather than rushing before the next public release.
PS:
>"Saturation" is not preserved
>"LoadOnDemand" is not preserved
>"UseMapping" is not preserved"

These three settings either are obsolete or are settings from GDI++ Helium version. They are not used anymore and do nothing here.

**Comment by sammilucia** [comment]
@snowie2000 yes I agree. I've marked it as a future version - not this release.

**Comment by iamr8** [comment]
I think it's better idea to create full wiki about sections and values, instead of have MacTuner in MacType

Otherwise, always experimental and new sections should have been supprted in INI, and after that, update MacTuner support and this will lead to confusions.

Isn't good ?
